pkgs/sagemath-standard/tox.ini

```diff
@@ -16,23 +16,20 @@
 #
 [tox]
 envlist =
-    #
-    # SUPPORTED ENVIRONMENTS:
-    #
-    # Build dependencies according to requirements.txt (all versions fixed).
-    # Use ONLY the wheels built and stored by the Sage distribution (no PyPI):
-    #
-    #   ./sage -sh -c '(cd pkgs/sagemath-standard && tox -v -v -v -e sagepython-sagewheels-nopypi)'
-    #
-    sagepython-sagewheels-nopypi,
-    #
     # Build and test without using the concrete dependencies specified by requirements.txt,
     # using the dependencies declared in pyproject.toml and setup.cfg (install-requires) only:
     # Still use ONLY the wheels built and stored by the Sage distribution (no PyPI).
     #
     #   ./sage -sh -c '(cd pkgs/sagemath-standard && tox -v -v -v -e sagepython-sagewheels-nopypi-norequirements)'
     #
-    sagepython-sagewheels-nopypi-norequirements,
+    sagepython-sagewheels-nopypi-norequirements
+    #
+    # OTHER SUPPORTED ENVIRONMENTS:
+    #
+    # Build dependencies according to requirements.txt (all versions fixed).
+    # Use ONLY the wheels built and stored by the Sage distribution (no PyPI):
+    #
+    #   ./sage -sh -c '(cd pkgs/sagemath-standard && tox -v -v -v -e sagepython-sagewheels-nopypi)'
```

Comment:
No `sagepython-sagewheels-nopypi` here?

Reply:
No, the `envlist` only lists the default environments, i.e., those that will be tested if one uses `tox` without the `-e` switch.

Reply:
OK. Thanks.